package.json

```diff
@@ -47,5 +47,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "devDependencies": {}
```

Comment:
Do we have devDependencies?

Reply:
Going forward we will want all non-production dependencies on dev, so not yet but on future refactoring